src/subscript/prtvol2csv/prtvol2csv.py

```diff
@@ -15,23 +15,34 @@
 from subscript import __version__, getLogger
 
 DESCRIPTION = """
-Extract reservoir volumes pr FIPNUM from Eclipse PRT files and dump to CSV.
+Extract in-place volumes per FIPNUM, or any FIP vector specified, from
+Eclipse PRT files and dump to CSV.
 
 If a yaml file is specified through options, it is possible to add columns
 with region and zone information to each FIPNUM. The YAML file must contain
-the keys "region2fipnum" and/or "zone2fipnum".
+the keys "region2fipnum" and/or "zone2fipnum". A YAML file can only be used together
+with FIPNUM, - not with any additional FIP vector.
 """
 
 CATEGORY = "utility.eclipse"
 
+
 EXAMPLES = """
 .. code-block:: console
 
-  FORWARD_MODEL PRTVOL2CSV(<DATAFILE>=<ECLBASE>, <REGIONS>=regions.yml, <DIR>=., <OUTPUTFILENAME>=simulator_volume_fipnum.csv)
+  FORWARD_MODEL PRTVOL2CSV(<DATAFILE>=<ECLBASE>, <REGIONS>=regions.yml, <FIPNAME>=FIPNUM, <DIR>=., <OUTPUTFILENAME>=simulator_volume_fipnum.csv)
 
 where ``ECLBASE`` is already defined in your ERT config, pointing to the Eclipse
 basename relative to ``RUNPATH`` and ``regions.yml`` is a YAML file defining
-the map from regions and/or zones to FIPNUM.
+the map from regions and/or zones to FIPNUM. The YAML file could be omitted in the
+FORWARD_MODEL and specified directly in the Webviz config file, if the REGIONS argument
+is not given a default value in the forward model job configuration.
+
+The ``FIPNAME`` argument is by default set to ``FIPNUM``, but any FIP-vector can be used. 
+Ensure the PRT file has volume reports for the additional FIP-vector. 
+
+The column name for any additional FIP-vector specified will be renamed to FIPNUM, as a temporary solution 
+to ensure the csv file will work with ``webviz-subsurface`` plugin ``VolumetricAnalysis``.
```

Comment:
Is this comment consistent with new rename argument?

Reply:
Good catch. Corrected now, reused a couple of lines from the .rst file.